CONTRIBUTING.md

```diff
@@ -3,6 +3,7 @@
 ✨ Thanks for contributing to **semantic-release**! ✨
 
 As a contributor, here are the guidelines we would like you to follow:
+
```

Comment:
Please remove the extra empty lines that were added in this PR

Reply:
Ok, I will remove them 

Unless I missed something it doesn't seem like XO formats markdown files? 

I still had prettier extension enabled witch formats the markdown in a consistent manner and Based on #653 is something you are trying to do. 

prettier uses [remark-parse](https://github.com/wooorm/remark/tree/master/packages/remark-parse) to format the markdown

Reply:
We don't format markdown with prettier because there is no way to enforce the format on the CI (with something like `xo` or `eslint`)

Reply:
Alright, all the changes are made. 

as for using prettier in you CI we do this on our ci using https://prettier.io/docs/en/cli.html#check 

we also use husky to setup git hooks to make sure staged files are formatted before pushing to remote and that the commit message follows our standards. So basically the only time someone pushes something that doesn't match our requirements is when they actively try to bypass the git hooks.

Reply:
At the time I configured the linting the `check` option didn't exists. I might revisit that at some point but not before we merge the `beta` branch to `master` as I don't want to add formatting change to the merge.

We don't use commit hook on purpose as we want to be able to push commits that might make violations while a PR is a work in progress. We only want to lint on the CI before merging a PR.
In addition we don't want to run the formatter on commit as it is confusing and error prone as it makes changes to the code without the developer control.